instances/widgets.treasury-factory.near/widget/pages/settings/members/MembersEditor.jsx

```diff
@@ -132,6 +132,16 @@ useEffect(() => {
   }
 }, [isTxnCreated, lastProposalId, treasuryDaoID]);
 
+const addProposalRole = (daoPolicy?.roles || [])?.find((role) =>
+  (role.permissions || []).some(
+    (i) =>
+      i === "transfer:AddProposal" ||
+      i === "*:AddProposal" ||
+      i === "transfer:*" ||
+      i === "*:*"
+  )
+)?.name;
```

Comment:
Not sure I understand this. Is the purpose to select a role that only has the "AddProposal" permission? This seems to suggest any role that has any of these permissions ( *:* )

Reply:
Yes you’re right, the thing for existing daos they usually have only one role, with all the permissions which is *:*, that’s why I added it.
As per our discussion in the Nearn group yesterday, I will create another PR to update the text when the role has voting permission as well, or do you think I should do it in this one itself?